src/Revenue/Migrations/CreateRevenueTable.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Give\Revenue\Migrations;
+
+use Give\Framework\Migrations\Contracts\Migration;
+use Give\Helpers\Table;
+
+class CreateRevenueTable extends Migration {
+	/**
+	 * @inheritDoc
+	 *
+	 * @since 2.9.0
+	 */
+	public static function id() {
+		return 'create_revenue_table';
+	}
+
+	/**
+	 * @inheritDoc
+	 *
+	 * @since 2.9.0
+	 */
+	public static function timestamp() {
+		return strtotime( '2019-09-16' );
+	}
+
+	/**
+	 * @inheritDoc
+	 *
+	 * @since 2.9.0
+	 */
+	public function run() {
+		global $wpdb;
+
+		$charset_collate = $wpdb->get_charset_collate();
+		$tableName       = Table::getName( 'give_revenue' );
```

Comment:
If we already have `$wpdb` as a dependency here, does it make sense to make use of `$wpdb` directly, as opposed to abstracting another dependency that does the same thing?

Reply:
Resolved in 0be28f8